src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,178 @@
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
+# Entries (p5 Template)
+
+[<img
+  src="../../assets/img/Entries/1729.png"
+  width=700
+  style="margin-left: 1em; margin-right: 0.5em"
+/>](../assets/img/Differences/squares.png)
+
+This visualizer allows you to step back and forth through a sequence, showing
+each entry as a written number.
+
+_This visualizer is meant to be used as a template for new visualizers based on
+the p5.js library. It includes explanatory comments and minimal examples of
+required and commonly used features._
+**/
+
+class P5VisualizerTemplate extends P5Visualizer {
+    // the name that appears in bundle card titles
+    static visualizationName = 'Entries (p5 Template)'
+
+    // parameters: top-level properties that the user can choose while creating
+    // the visualizer bundle
+    stepSize = 1 // the default value of this parameter is zero
+
+    /** md
+## Parameters
```

Comment:
And here you can point out that every visualizer has documentation of all parameters.  (Am I interpreting the spirit of this file correctly?  The code and markdown are supposed to be a straight-up working example, but the comments are information you will want as you learn to make visualizers.)

Reply:
Maybe the spirit is to include _instructions_ in the documentation and not in comments in the example, in which case you could leave these out and just add a section on documentation in the doc file for Making A Visualizer.